src/index.js

```diff
@@ -224,9 +228,11 @@ async function jsPDFInvoiceTemplate(props) {
         },
       },
       total: {
-        col1: props.data?.total?.col1 || "",
-        col2: props.data?.total?.col2 || "",
-        col3: props.data?.total?.col3 || "",
+        col1: props.data?.total?.col1 || "", // Total label
```

Comment:
How big of a change it would be to change keys `col1` col2` to something more meaningful? Its quite hard to understand

Reply:
Will take sometime. But can be done in a separate PR.